### ISSUE TITLE
casm update does not read in properties.calc.json file created be vasp.relax.report

### ISSUE THREAD

**Issue by mxgrabowski** [prisms-center/CASMcode#52]
I am trying to import calculations with larger supercells (108 atoms with Ni-Re-Va) to my project. I used vasp.relax.report with POSCAR,OUTCAR,OSZICAR and vasprun.xml which created me a properties.calc.json. However, when I am trying to update with 'casm update', I get the error: 

Reading calculation data... 

***************************

Working on /home/users/grabomqf/DFT/vasp5.4/ClusterExpansion/ternary/training_data/SCEL108_6_6_3_3_3_0/0/calctype.default/properties.calc.json

Error: Unable to read properties.calc.json
get_value< string > called on integer Value

A few days ago, I updated my casm version from 0.2.0+b20bd4e.changes (master branch in september 16) to the version 0.2.1+e413032 (master branch from 4/11/2017). I set up a new project for this CE so that the files created from vasp.relax.report should be from the same version as the project is set up.

I am attaching the prim.json and one of the properties.calc.json
[properties_prim.gz.zip](https://github.com/prisms-center/CASMcode/files/953494/properties_prim.gz.zip)

Thank you for the help,
Maximilian

**Comment by bpuchala** [comment]
The `"atom_type"` field in the `properties.calc.json` file has been printed incorrectly with numbers instead of atom names. I don't have your calculation files to try to reproduce the error, but I haven't been able to reproduce this error in my test project. A couple things to check:

1) Does using `"casm-calc --report"` to print `properties.calc.json` files work?
1) Is your SPECIES file OK?

**Comment by mxgrabowski** [comment]
1. Using `"casm-calc --report"` does work to print `properties.calc.json`, but it will still print the `"atom_type"` as numbers.
2. I didn't have a `SPECIES` file, so it doen't seem to use this to create `properties.calc.json`? I created one, but this didn't change anything.

I checked my `POSCAR` file which I used to create the `properties.calc.json`. I realized, I didn't have the line which identifies the species in the configuration.
I.e.:
Supercell of Ni
1.00000000
                      10.53074840       0.00000000       0.00000000
                       0.00000000      10.53074840       0.00000000
                       0.00000000       0.00000000      10.53074840
2 105
Direct
  0.1666666666666667  0.1666666666666667  0.0000000000000000
  0.1666666666666667  0.0000000000000000  0.1666666666666667

instead of
Supercell of Ni
1.00000000
                      10.53074840       0.00000000       0.00000000
                       0.00000000      10.53074840       0.00000000
                       0.00000000       0.00000000      10.53074840
Re Ni
2 105
Direct
  0.1666666666666667  0.1666666666666667  0.0000000000000000
  0.1666666666666667  0.0000000000000000  0.1666666666666667

This made it possible to create the `properties.calc.json` with the correct string definition of the species. So we should be aware, that we need Vasp5 POSCARs to create the `properties.calc.json` files.
